static/src/javascripts-legacy/projects/common/modules/commercial/membership-engagement-banner.js

```diff
@@ -34,12 +35,31 @@ define([
                  $,
                  defaults,
                  find,
-                 svgs) {
+                 svgs,
+                 fetch) {
 
         // change messageCode to force redisplay of the message to users who already closed it.
         // messageCode is also consumed by .../test/javascripts/spec/common/commercial/membership-engagement-banner.spec.js
         var messageCode = 'engagement-banner-2017-01-11';
 
+        var SECONDARY_BUTTON_SELECTOR = '.secondary';
+
+        //Remind me form selectors
+        var REMIND_ME_FORM_SELECTOR = '.membership__remind-me-form';
```

Comment:
> The *_SELECTOR are constant

@SiAdcock are we using const yet? I look forward to the day when all variables are constant 😜 

> Does frontend use camelCase or snake_case for variables

My pref? camelCase unless you are DECARING_GLOBALS. I would follow the convention of `var messageCode` to reduce the use of caps

other places in frontend sometimes use `el` in the name... would `remindMeTextFieldEl` work?

However, if these constants are only being used once, could we scope them within the function that will use them? Or just pass the string into the function? I find the following much easier to grok:

`document.querySelector('.membership__engagement-text-field');`